docker-entry.sh

```diff
@@ -15,5 +15,4 @@ if [ -z ${TRAVIS-} ]; then
 fi
 
 # use the wait-for-it util to ensure the server is running before invoking Tox
-util/wait-for-it.sh ${REDIS_MASTER} -- tox -- --redis-url=redis://"${REDIS_MASTER}"/9
-
+util/wait-for-it.sh ${REDIS_MASTER} -- tox -- --redis-url=redis://"${REDIS_MASTER}"/9 --ssl-port=${REDIS_MASTER_SSL_PORT} --docker
```

Comment:
Not sure about this approach.

We should test both the SSL and non-SSL connections. We can run both on the same master, but using different ports.